remind-bindings.el

```diff
@@ -223,6 +222,7 @@
 
 (defun remind-bindings-sidebuffer-make (hashtable)
   "Populate a sidebuffer with a HASHTABLE of bindings."
+  (require 'org)
```

Comment:
wow you can lazy load libraries like this?

Reply:
I read the whole package and org was not needed at compile time.
Libraries that are needed only when executing a function can be loaded in this way.
This technique is also used in Elisp that comes with Emacs attached.